pkg/plugins/golang/v2/scaffolds/internal/templates/dockerfile.go

```
@@ -39,7 +39,7 @@ func (f *Dockerfile) SetTemplateDefaults() error {
 }
 
 const dockerfileTemplate = `# Build the manager binary
-FROM golang:1.13 as builder
+FROM golang:1.16 as builder
```

Leave go/v2 as 1.13. This plugin is frozen.
```suggestion
FROM golang:1.13 as builder
```